themes/bootstrap3/templates/RecordTab/holdingsils/extended.phtml

```diff
@@ -29,7 +29,16 @@
           <?php else: ?>
             <?php if ($holding['availability']): ?>
               <?php /* Begin Available Items (Holds) */ ?>
-              <span class="text-success"><?=$this->transEsc("Available")?><link property="availability" href="http://schema.org/InStock"></span>
+              <?php
+                if ((int)$holding['availability'] === \VuFind\ILS\Logic\ItemStatus::STATUS_UNCERTAIN) {
+                  $statusClassEsc = 'text-warning';
+                  $status = $holding['status'];
+                } else {
+                  $statusClassEsc = 'text-success';
+                  $status = is_int($holding['availability']) && !empty($holding['status']) ? $holding['status'] : 'Available';
+                }
+              ?>
+              <span class="<?=$statusClassEsc?>"><?=$this->transEsc($status)?><link property="availability" href="http://schema.org/InStock"></span>
```

Comment:
Does uncertain status require a different schema.org property?

Reply:
I guess it depends on how you think about it, but I changed it to `LimitedAvailability` since that should cover most cases.